### PR TITLE
:sparkles: Create a new contact form for Equinor pension

### PR DESCRIPTION
**Pull request by VarunVAshrit**

### CODE REVIEW

File: web/pages/api/forms/service-now-pension.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { PensionFormCatalogType } from '../../../types'
+import { sendRequestToServiceNow } from './service-now-base'
+import { validateFormRequest } from './validateFormRequest'
+
+const getCatalogIdentifier = (catalogType: PensionFormCatalogType | null) => {
+    console.log(catalogType)
```
Comment (padms) [inline web/pages/api/forms/service-now-pension.ts:7]: Clean up this log

**Review by padms — CHANGES_REQUESTED** [review]
Minor clean up to be done. Otherwise looks good to me.

File: web/pages/api/forms/service-now-pension.ts (hunk 1 above)
Comment (VarunVAshrit) [reply web/pages/api/forms/service-now-pension.ts]: Done

**Review by padms — APPROVED** [review]

**Review by padms — CHANGES_REQUESTED** [review]
There is something missing with the styles, the space around the form fields do not resemble the other forms

**Comment by BorghildSelle** [comment]
This form will need to be updated to look similar with the other forms as in the #2510, 
so will try to get the other PR into main so this branch can merge in main and be updated

**Pull request merged**